pkg/ovalutil/test.go

```diff
@@ -0,0 +1,34 @@
+package ovalutil
+
+import (
+	"fmt"
+
+	"github.com/quay/goval-parser/oval"
+)
+
+func TestLookup(root *oval.Root, ref string, f func(kind string) bool) (oval.Test, error) {
+	kind, i, err := root.Tests.Lookup(ref)
+	if err != nil {
+		return nil, err
+	}
+	if ok := f(kind); !ok {
```

Comment:
should this always be ran? you can check a function for nil, then the caller doesn't need to spend more text on a no-op function.